overlord/managers_test.go

```diff
@@ -14435,13 +14441,29 @@ func (s *mgrsSuite) TestConnectionDurabilityDuringRefreshes(c *C) {
 	mockServer := s.mockStore(c)
 	defer mockServer.Close()
 
+	canAutoRefreshCalls := 0
+	snapstate.CanAutoRefresh = func(*state.State) (bool, error) {
```

Comment:
do we need to undo this?

Reply:
no, it's set to nil in suite setup (which ten causes a panic if a respective test doesn't mock it again)